src/pages/ExamsPage/components/AttemptList.jsx

```diff
@@ -2,10 +2,16 @@ import PropTypes from 'prop-types';
 import { DataTable } from '@edx/paragon';
 import { useIntl } from '@edx/frontend-platform/i18n';
 import ResetExamAttemptButton from './ResetExamAttemptButton';
+import ReviewExamAttemptButton from './ReviewExamAttemptButton';
 
-const capitalizeFirstLetter = (string) => string.charAt(0).toUpperCase() + string.slice(1);
+const cleanString = (string) => {
+  // Remove underscores, capitalize the first letter of each word, and return as a single string.
+  return string.split("_").map(string => string.charAt(0).toUpperCase() + string.slice(1)).join(" ");
```

Comment:
if we're going down the path of formatting these we should probably just create a map between the backend value and UI labels since this method isn't internationalizable (if that's a word...)

Reply:
I have improved this by adding this mapping in data/,constants.js